src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs

```diff
@@ -406,7 +406,14 @@ void UpdateFooter()
 			}
 
 			if (footer == null)
```

Comment:
From https://github.com/xamarin/Xamarin.Forms/pull/15306/files

Reply:
Wait with this one, I might need to revert it as it causes a regression

Reply:
Reverted.